config/routes.rb

```diff
@@ -168,7 +168,9 @@
 
           resource :reporting_type, only: %i[edit update]
 
-          resource :constraints, only: %i[show update]
+          resources :constraints, only: %i[index create destroy] do
+            patch :update, on: :collection
+          end
```

Comment:
Why the change to `resources`?

Reply:
Because I wanted the :id on the create and destroy method?